PusherMainView/PusherMainViewTests/PusherStoreSpec.swift

```diff
@@ -81,6 +81,14 @@ final class PusherStoreSpec: QuickSpec {
                                             type: .token(keyID: "keyID",
                                                          teamID: "teamID",
                                                          p8: "p8"))
+        let failedAPNSMock = APNSPusherMock(result: .failure(NSError(domain: "com.pusher.error",
+                                                                     code: 400,
+                                                                     userInfo: nil)),
+                                            type: .token(keyID: "keyID",
+                                                         teamID: "teamID",
+                                                         p8: "p8"))
+        let validJSONstring = #"{"":""}"#
```

Comment:
@imaginaris It's interesting but it's not the topic of this PR.
Here it handles an issue that users may have when they enter an invalid JSON payload. It can happen with QA.

Can you create a ticket about verifying the APNS payload model?

Reply:
Ok, sure. SDKCF-5525